src/compiler/program.ts

```diff
@@ -119,16 +122,19 @@ module ts {
         var diagnostics = createDiagnosticCollection();
         var seenNoDefaultLib = options.noLib;
         var commonSourceDirectory: string;
+        var diagnosticsProducingTypeChecker: TypeChecker;
+        var noDiagnosticsTypeChecker: TypeChecker;
+
         host = host || createCompilerHost(options);
 
+        var start = new Date().getTime();
         forEach(rootNames, name => processRootFile(name, false));
         if (!seenNoDefaultLib) {
             processRootFile(host.getDefaultLibFileName(options), true);
         }
-        verifyCompilerOptions();
+        programTime += new Date().getTime() - start;

```

Comment:
Why is programTime set here?  It isn't measuring the time to verifyCompilerOptions it seems (and that's part of producing the program).